conf/openlibrary-docker.yml

```diff
@@ -0,0 +1,173 @@
+# Open Library Docker (dev) configuration
```

Comment:
The removal of unnecessary/outdated keys and whatnot should also be reflected in the original config file, for consistency (although it will be deleted once we switch to docker :P)